packages/flutter_tools/lib/src/commands/channel.dart

```diff
@@ -65,32 +65,55 @@ class ChannelCommand extends FlutterCommand {
       <String>['git', 'branch', '-r'],
       workingDirectory: Cache.flutterRoot,
       mapFunction: (String line) {
```

Comment:
Does this work: `mapFunction: rawOutput.add,`

Reply:
Nope. because return type of `mapFunction` is `String` and `rawOutput.add` is `void`